tests/functional/views/admin_test.py

```diff
@@ -0,0 +1,5 @@
+def test_admin_authentication_redirects_to_google(app):
```

Comment:
Veryyyy light touch functional testing for this. Pretty much a sanity check.

I don't think this warrants the effort of simulating the whole oauth flow with google but this is better than nothing.

Reply:
Yeah, makes sense. You've got to be careful not to create more problems than you're solving when writing tests. Test code maintenance is a real issue. And functional tests are particularly prone to being really hard to understand and maintain because so much code is being executed under them. Best to keep it simple :+1: